release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -100,6 +100,12 @@ For more information, see xref:../installing/installing_aws/installing-aws-secre
 
 //For more information, see ../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installing-nutanix-installer-provisioned[Installing a cluster on Nutanix].
 
+[id="ocp-4-11-azure-ultrassd"]
```

Comment:
Passive voice for "are installing"

Do you want to remove that URL? I take it being commented that you will uncomment this link when it goes live?

Reply:
Thanks Eric - I believe I resolved the passive voice. I also uncommented and fixed the URL. The content that this release note addresses has not been merged yet, but the link and anchor are correct for when it is merged.